### PR TITLE
Referee relationship and safeguarding

### PR DESCRIPTION
**Pull request by paulrobertlloyd**
## Relationship

Confirmed:
Screenshot 2020-03-02 at 15 02 23

Amended:
Screenshot 2020-03-02 at 15 02 12

## Safeguarding

No corcerns:
Screenshot 2020-03-02 at 14 38 55

Unknown concerns:
Screenshot 2020-03-02 at 14 38 42

Some concerns:
Screenshot 2020-03-02 at 14 38 33

### CODE REVIEW

**Comment by adamsilver** [comment]
I don't recommend having multiple pieces of information per row because I think it's harder to scan—users have to look in the headings as well as the value cells to find what they're looking for.

This also makes it more difficult to use for screen reader users as they would have to move into the value cell (with a row heading of ‘relationship’) and move past the value in order to find out quickly whether the referee confirms the relationship as the candidate described—and if not, why.

Putting information inside the cells also deviates from the base pattern.

Given this, I'd be looking for strong proof this has a major benefit to certain users.

@laurahermionetennant92 @paulrobertlloyd

File: app/views/_includes/application/reference.njk [hunk 1]
```diff
@@ -17,6 +17,40 @@
...
+  {% endif %}
+{% endset %}
+{% set safeguardingHtml -%}
+  {%- if reference.safeguarding.response == "yes" -%}
+    {{ govukInsetText({
+      html: reference.safeguarding.concerns | nl2br,
+      classes: "govuk-!-margin-top-0 govuk-!-padding-top-0 govuk-!-padding-bottom-0"
+    }) }}
+  {%- elif reference.safeguarding.response == "no" -%}
+    <p class="govuk-body">Referee has none</p>
```
Comment (fofr) [inline app/views/_includes/application/reference.njk]:
```suggestion
    <p class="govuk-body">Referee has no concerns</p>
```

Comment (adamsilver) [reply app/views/_includes/application/reference.njk]: Again here I'd suggest:

Does the referee have concerns about safeguarding? Yes
What are the referee's concerns? [Blah blah blah]

**Comment by laurahermionetennant92** [comment]
Will review this on Tues @paulrobertlloyd (thanks for fixing Github issue)

**Comment by laurahermionetennant92** [comment]
@paulrobertlloyd Soz can't do this:

Hint text plus straight 'No' 'Don't know' or Yes plus reason (with grey bar?)
Screenshot 2020-03-03 at 09 40 45

**Comment by laurahermionetennant92** [comment]
Not convinced how clear distinction is between candidate's words and referee's words. Can we try putting the reference in a grey box?
Also I tried to take the summary of the relationship out of quotation marks but it bust when I did:
Screenshot 2020-03-03 at 09 44 31

**Comment by laurahermionetennant92** [comment]
'Yes'
Screenshot 2020-03-03 at 09 50 09
 followed by concern in grey box?

**Comment by laurahermionetennant92** [comment]
If the answer to safeguarding is 'Don't know' just display that

**Comment by laurahermionetennant92** [comment]
@paulrobertlloyd @adamsilver 

Have written some content so we can follow 'Question: answer' pattern on single row. 

**Relationship between candidate and referee: content

Confirmed by referee? Yes

Amended by referee: content

Does referee know of a reason why this candidate should not work with children? No/Don't know/content

Reference: content**

@paulrobertlloyd
a) I wanted to be more transparent to provider about the specific question we ask the referee
b) I think it's not currently clear what is candidate input and what is referee input – formatting it like this (potentially) removes need for grey bar or icon 
c) addresses Adam's concern about parsing more than 1 chunk of info per row.

**Comment by paulrobertlloyd** [comment]
Updated how we display these values.

**Example 1**: Amended relationship
Screenshot 2020-03-03 at 11 57 27

**Examples 2**: Safeguarding concern
Screenshot 2020-03-03 at 11 57 40

**Pull request merged**